resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Bad Request",
+          "errors": [
+            {
+              "code": "MISSING_FIELD",
+              "message": "This field is required",
+              "path": "/lastName"
+            },
+            {
+              "code": "INVALID_DATE",
+              "message": "Date is invalid",
+              "path": "/dateOfBirth"
+            },
+            {
+              "code": "INVALID_FORMAT",
+              "message": "Invalid format has been used",
+              "path": "/investorNINO"
+            },
+            {
+              "code": "INVALID_DATA_TYPE",
+              "message": "Invalid data type has been used",
+              "path": "/firstName"
+            }
+          ]
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing investor details which do not match HMRC’s records</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorNINO": "AA111111A",
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "dateOfBirth": "1985-03-25"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_NOT_FOUND",
+          "message": "The investor details given do not match with HMRC’s records"
+        }
+        </pre>
+                     </td>
+                </tr>
+                <tr>
+                   <td><p>Request with an invalid 'Accept' header</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br><br>Accept: application/vnd.hmrc.1.0</p></td>
+                   <td>
+        <pre class="code--block">
+        {
+          "investorNINO": "AA123456A",
+          "firstName": "First Name",
+          "lastName": "Last Name",
+          "dateOfBirth": "1985-03-25"
+        }
+        </pre>
+                   </td>
+                   <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                   </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing a pre-existing investor’s details</p> <p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks